lib/perl/Genome/Model/Tools/Picard/ConcatenateMetricsFiles.pm

```diff
@@ -0,0 +1,93 @@
+package Genome::Model::Tools::Picard::ConcatenateMetricsFiles;
+
+use strict;
+use warnings;
+
+use Genome;
+use Data::Dumper;
+
+class Genome::Model::Tools::Picard::ConcatenateMetricsFiles {
+    is  => 'Command::V2',
+    has => [
+        metrics_files   => {
+            is  => 'String',
+            doc => 'An input Picard metrics file',
+            is_many => 1,
+        },
+        output_file  => {
+            is  => 'String',
+            doc => 'The output concatenated metrics file',
+        },
+        subclass_name => {
+            is => 'String',
+            is_optional => 1,
+            valid_values => ['CollectRnaSeqMetrics', 'GenotypeConcordance', 'MarkDuplicates'],
```

Comment:
There are a few other Picard subclasses that have `_metric_header_as_key` defined:
* `CollectAlignmentSummaryMetrics`
* `CollectGcBiasMetrics`
* `CollectInsertSizeMetrics`

I'm not sure if this tool would be useful for any of them?